state_machine/applications/example/Tasks/battery.py

```diff
@@ -0,0 +1,11 @@
+# check for low battery condition
+
+from lib.template import Task
```

Comment:
It looks like you updated this import from `lib.template_task` to `lib.template` without changing the name of the file. The filename to change would be found at state_machine/frame/lib/template_task.py

Reply:
Hello, got the feedback. Working on resolving it.